pages/place/update/views/confirm.jsx

```diff
@@ -60,6 +61,14 @@ const Confirm = ({
             )
             : <Diff formatters={formatters} />
         }
+        {
+          values && values.comments && (
+            <Fragment>
+              <h2><Snippet>fields.comments.label</Snippet></h2>
+              <p>{ values && values.comments }</p>
```

Comment:
This second check for `values &&` is redundant.

Reply:
Do we definitely want to hide this content if no comment is provided? Or should there be some kind of "no comment provided" message. @pcuk1979?